### PR TITLE
GSG Further edits - Part 1

### PR DESCRIPTION
**Pull request by Loquacity**
~~_**MERGE https://github.com/timescale/docs/pull/2602 FIRST!!**_~~

# Description

* Services: make connection more prominent
* Queries: Add time_bucket query
* Aggregation: title change and remove average procedure

# Links

Fixes https://github.com/timescale/docs-private/issues/170

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: getting-started/services.md [hunk 1]
```diff
@@ -31,23 +31,37 @@ connect to it from the command prompt on your local machine. You can then use
 `psql` to create tables and add data directly into your database.
 
 In this section, you sign up for a Timescale account, create a service, and
-connect to it from your local machine using `psql`.
+connect to it from your local machine using `psql`. Don't forget to download the
+cheatsheet when you create your service, it gives you important information that
```
Comment (Rajakavitha1) [inline getting-started/services.md:35]:
```suggestion
cheatsheet when you create your service, it contains important information that
```

**Review by Rajakavitha1 — APPROVED** [review]
LGTM

File: getting-started/queries.md [hunk 2]
```diff
@@ -159,5 +159,63 @@ For more information about these functions, see the API documentation for
 
 </Procedure>
 
-[first]: /api/:currentVersion:/hyperfunctions/first
-[last]: /api/:currentVersion:/hyperfunctions/last
+## Use time buckets to get values
+
+To make it easier to look at numbers over different time ranges, you can use the
+Timescale `time_bucket` function. Time buckets are used to group data, so that
```
Comment (noctarius) [inline getting-started/queries.md:165]: Maybe add a sentence such as:
_Time buckets represent a specific point in time according to the bucket definition. All timestamps between the time bucket start and the selected length will be truncated down to the bucket timestamp. Time buckets are used to group ..._

File: getting-started/queries.md [hunk 3]
```diff
@@ -159,5 +159,63 @@ For more information about these functions, see the API documentation for
...
+you can perform calculations over different time periods.
+
+In this section, you use the same query as the previous section to find the
+`first` and `last` values, but start by organizing the data into 1-hour time
+buckets.
```
Comment (noctarius) [inline getting-started/queries.md:170]: `first` and `last` aggregations return the first and last seen value for the current time bucket grouping and sorting accordingly.

**Review by noctarius — APPROVED** [review]

**Comment by Loquacity** [comment]
This is ready to merge now, so happy for you to review please! 💖

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!!!

File: getting-started/aggregation.md [hunk 4]
```diff
@@ -31,10 +31,10 @@ records that you need to look up to perform your query.
 
 <CaggsTypes />
 
-In this section, you create a simple aggregation by finding the average trade
-price over a week. Then, you create a materialized
-view, transform it into a continuous aggregate, and query it for more
-information about the trading data.
+In this section, you create a materialized view, transform it into a continuous
```
Comment (noctarius) [inline getting-started/aggregation.md:34]: I know the text mentioned it before, and it's sad I just realize it now, but you do not create a materialized view and transform it into a continuous aggregate. You create a continuous aggregate through a similar syntax to a materialized view in PostgreSQL :-)

**Pull request merged**